src/window/context_settings.rs

```diff
@@ -42,6 +41,21 @@ pub const CONTEXT_DEBUG: u32 = 1 << 2;
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Debug, Copy)]
 pub struct ContextSettings(pub ffi::sfContextSettings);
 
+impl ContextSettings {
+    /// Creates a new ContextSettings with all parameters 
+    /// default, except `antialiasing_level`
+    pub fn new_with_antialiasing_level(aa_level: u32) -> ContextSettings {
```

Comment:
I don't think adding a `new_with_foo` function for every property we want to set is a composable solution.

We could instead support the [builder pattern](https://doc.rust-lang.org/book/method-syntax.html#builder-pattern), so the user could do e.g. `let ctx = ContextSettings::default().antialiasing(8);`.